src/Workspaces/Core/Portable/CodeGeneration/AbstractCodeGenerationService.cs

```diff
@@ -262,7 +262,10 @@ protected static T Cast<T>(object value)
                 // Metadata as source generates complete declarations and doesn't modify
```

Comment:
This code isn't specific to Metadata As Source, right? Would this regress other scenarios?